tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -234,6 +236,24 @@ public void B2CLocalAccountAcquireTokenTest()
             _mobileTestHelper.B2CLocalAccountAcquireTokenSilentTest(xamarinController, LabUserHelper.GetB2CLocalAccount(), true);
         }
 
+        /// <summary>
+        /// B2C acquire token with ROPC
+        /// </summary>
+        [Test]
+        public void B2CROPCAcquireTokenTest()
+        {
+            _mobileTestHelper.B2CROPCLocalAccountAcquireTokenTestHelper(xamarinController, LabUserHelper.GetB2CLocalAccount());
```

Comment:
We should most definitely not have an Android test for this. This can easily be tested using a Headless integration test. It doesn't even require Selenium.

Reply:
right...#guru. forgot about the headless tests. will do. thanks.

---
In reply to: [271517887](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1034#discussion_r271517887) [](ancestors = 271517887)